### PR TITLE
(cargo-release) version 0.3.0

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #176** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/176" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/176?utm_source=stack-comment).

**Review by jcape — APPROVED** [review]

**Review by awygle — APPROVED** [review]

**Comment by nick-mobilecoin** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/176?utm_source=stack-comment).


Job Reference: 15aK1CF6xb9qdoenZPPg

**Pull request merged**
